### PR TITLE
Add FAQ

### PR DESCRIPTION
**Pull request by zeusdeux**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+No, the iframe sandboxing restrictions will prevent you from doing so.
+
+## May I open regular javascript popups from an extension?
+
+Yes, you can use `window.open()` and links with `target="_blank"`
+
+## Is it mandatory to host my extensions on HTTPS?
+
+We do not allow you to reference non-HTTPS URLs (like `http://`) with the exception of URLs starting with `http://localhost` for development purposes.
+If you serve the extension from `localhost` you need to disable [mixed content protection in the browser](https://support.mozilla.org/en-US/kb/mixed-content-blocking-firefox, https://support.google.com/chrome/answer/1342714).
```
Comment (hlabas) [inline FAQ.md]: Double linked here leading to a broken link

File: FAQ.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## Is CORS required?
+
+As the extension is run in a sandboxed Iframe it receives the origin “null”.
+Therefore, if you’re using custom web fonts, things don't work as expected as they don’t play well with sandboxed iframes at the moment.
+
+To overcome this, you can configure your servers to send the following headers:
+`Access-Control-Allow-Origin` "null" or  `Access-Control-Allow-Origin` "*"
```
Comment (hlabas) [inline FAQ.md]: nit: I'd include the "null" and "*" in the formatting back quotes

File: FAQ.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+## What happens when deploying a broken extension?
+
+If the URL defined in your `extension.json` is invalid or contains errors, then the extension will either not show up or show as broken.
+
+## How do you debug extensions?
+
+Most of [our examples](https://github.com/contentful/bundles/tree/master/extensions) demonstrate how to deploy an extension running on a local web server, so you can use your browser console to debug it the same way you debug any other web application.
```
Comment (hlabas) [inline FAQ.md]: My bad, this link has changed to https://github.com/contentful/extensions/tree/master/samples

File: FAQ.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+This is not possible right now, but we’re working on adding this possibility in the future.
+
+## What happens with the content once a extension is removed?
+
+The content is independant from the extension that is used to edit it. When you remove an extension the content is still there and you can choose another appearance from the field’s settings.
+
+## Can the extension destroy my content?
+
+In theory yes, since they have access to all the content the user that is running the extension has. Make sure you implement safeguards to avoid losing your content.
```
Comment (hlabas) [inline FAQ.md]: Maybe "to avoid removing your content inadvertently"

File: FAQ.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+## What are the core components and how do they differ from extensions?
+
+Core components are what we provide as a part of the Contentful Web Application. Our core components are currently leveraging the same API that the UI Extension SDK provides, and we are planning to open source them so you can actually create custom extensions based on them.
+
+## How do I migrate to the new version of the UI Extensions SDK?
+
+If you were building extensions as a part of the beta program from March to July 2016, then you will need to update your extensions for them to continue working. Since the term 'widget' has been deprecated in favor of 'extensions', here are the parts that you need to update:
```
Comment (hlabas) [inline FAQ.md]: Maybe adding an internal link to the migration question would be nice

**Comment by hlabas** [comment]
A couple of minor changes but LGTM after that.

File: FAQ.md (hunk 4 above)
Comment (zeusdeux) [inline FAQ.md]: Imho, "losing your content" sounds more impactful. And that might be the right tone for this message.

Comment (hlabas) [inline FAQ.md]: SGTM, let's keep this then

**Pull request merged**